modules/migration-setting-up-target-cluster-to-accept-source-dns-domain.adoc

```diff
@@ -15,12 +16,12 @@ For both non-secure HTTP access and secure HTTPS access, perform the following s
 +
 [source,terminal]
 ----
-$ oc expose svc app1-svc --hostname app1.apps.ocp3.example.com \
+$ oc expose svc app1-svc --hostname app1.apps.source.example.com \
  -n app1-namespace
```

Comment:
user-defined variables appear in <>. Use them for both.

Second is <namespace> or <application_namespace>

Also, using callouts is recommended for this kind of code example so that the user does not have to figure out what is required by reading your example.